data_steward/tools/generate_ext_tables.py

```diff
@@ -160,7 +159,7 @@ def create_and_populate_source_mapping_table(project_id, dataset_id):
                                    SITE_MAPPING_FIELDS,
```

Comment:
I would suggest changing lines 134 and 135 to using f-strings too.  I can't comment on that line directly because it hasn't been changed in this PR.

```
        bq_insert_list.append(f'("{hpo_rdr_item[0]}", "{hpo_rdr_item[1]}")')